frontend/src/metabase/parameters/components/ValuesSourceModal/ValuesSourceModal.unit.spec.tsx

```diff
@@ -368,34 +380,41 @@ describe("ValuesSourceModal", () => {
 interface SetupOpts {
   parameter?: UiParameter;
   parameterValues?: ParameterValues;
+  databases?: Database[];
+  collections?: Collection[];
   cards?: Card[];
-  hasDataAccess?: boolean;
+  hasCollectionAccess?: boolean;
   hasParameterValuesError?: boolean;
 }
 
-const setup = ({
+const setup = async ({
   parameter = createMockUiParameter(),
   parameterValues = createMockParameterValues(),
+  databases = [createMockDatabase()],
+  collections = [createMockCollection(ROOT_COLLECTION)],
   cards = [],
-  hasDataAccess = true,
+  hasCollectionAccess = true,
```

Comment:
Data access is a different thing, so lets call it properly here to avoid confusion